lib/common.js

```diff
@@ -154,11 +154,13 @@ function stringifyRequest(options, body) {
   let { port } = options
   // TODO-coverage: Add a test to cover the missing condition, or remove if
   // not reachable.
-  if (!port) port = options.proto == 'https' ? '443' : '80'
+  // The step seems useless considering we don't show the port for standard 80/443 anyway.
+  // The net affect this has is to default the port to 80 when the proto *is not* http. (which probably isn't right)
```

Comment:
This one line of the comment doesn't make sense to me. Could you explain it?

Reply:
So this comment just speaks to what would be different if this whole line was removed. 

If the proto is 'https' then the port becomes '443' and that combo clears the port in the next logic block.
If the proto is 'http' then the port becomes '80' and that combo clears the port in the next logic block.
**However**, if the proto is not 'http' or 'https' (which I don't think is possible in reality), then the port is set to '80' and that value is on the final result of this function. 

Following the flow of how `proto` can be set on the `options` object, I *think* it's impossible for the value not to be `http` or `https`. And if that's the case, this line can just be deleted as it serves no purpose.

Reply:
Gotcha. I suggested an update.